proto/osmosis/incentives/gauge.proto

```diff
@@ -9,33 +9,41 @@ import "osmosis/lockup/lock.proto";
 
 option go_package = "github.com/osmosis-labs/osmosis/v7/x/incentives/types";
 
+// Gauge is a struct that works as a single unit for each distribution of
+// incentives defined by x/incentives module.
 message Gauge {
-  // unique ID of a Gauge
+  // ID is the unique id of the gauge.
+  // The ID of the gauge is decided upon gauge creation, incrementing by 1 for
+  // every gauge.
   uint64 id = 1;
-  // flag to show if it's perpetual or multi-epoch
-  // distribution incentives by third party
+  // IsPerpetual defines whether the gauge is perpetual or not.
+  // A non perpetual gauge distributes the incentive tokens equally per epoch
+  // during the gauge is in an active period.
+  // A perpetual gauge distributes all tokens at a single distribution, mainly
+  // used to distribute minted osmo to lp token stakers.
   bool is_perpetual = 2;
-  // Rewards are distributed to lockups that are are returned by at least one of
-  // these queries
+  // Distribute To is the lock query condition that the gauge reward are to be
```

Comment:
I'm not sure if the decision was to avoid merging the var name words? I just noticed some of them still have spaces
```suggestion
  // DistributeTo is the lock query condition that the gauge reward are to be
```

Reply:
I think we should merge var name words